tests/scr/sysbox

```diff
@@ -28,45 +28,163 @@ function retry() {
   false
 }
 
-# Increase default inotify resources to meet sys container demands.
-sysctl -w fs.inotify.max_queued_events=1048576
-sysctl -w fs.inotify.max_user_watches=1048576
-sysctl -w fs.inotify.max_user_instances=1048576
-
-# Increase default keyring resources to meet sys container demands.
-# For a k8s cluster:
-# maxkeys = 35 + (workers * 23) + (2 * pods)
-# maxbytes = 20 bytes * maxkeys
-sysctl -w kernel.keys.maxkeys=20000
-sysctl -w kernel.keys.maxbytes=$((20*20000))
-
-sleep 1
-
-# start sysbox-mgr
-sysbox-mgr --log /dev/stdout > /var/log/sysbox-mgr.log 2>&1 &
-RES=$(retry 10 1 grep -q Ready /var/log/sysbox-mgr.log)
-
-if [ $? -ne 0 ]; then
-  printf "\nsysbox-mgr failed to start. Here is the log file:\n"
-  cat /var/log/sysbox-mgr.log
-  exit 1
-fi
-
-# start sysbox-fs
-#
-# Note: inside the test container we must ignore sysbox-fs handler
-# errors when accessing procfs and sysfs, because some procfs and
-# sysfs resources are not writeable from within the test container
-# (see sysbox issue #538).
-mkdir -p /var/lib/sysboxfs
-sysbox-fs --ignore-handler-errors --log /dev/stdout > /var/log/sysbox-fs.log 2>&1 &
-RES=$(retry 10 1 grep -q Initiating /var/log/sysbox-fs.log)
-
-if [ $? -ne 0 ]; then
-  printf "\nsysbox-fs failed to start. Here is the log file:\n"
-  cat /var/log/sysbox-fs.log
-  exit 1
-fi
-
-
-exit 0
+# Ensure that kernel-modules expected by system-level apps (running within sys
```

Comment:
Rather than duplicating all this code, maybe we can get rid of `tests/scr/sysbox`, and instead use `scr/sysbox` but pass it a flag that tells it it's running in a test environment. If this flag is set, then we start sysbox-fs with "--ignore-handler-errors".

Reply:
Agree. Done.